### ISSUE TITLE
Parts of this Website are not Safe browseralert

### ISSUE THREAD

**Issue by scholzling** [fablabcb/fablabcb.github.io#1]
Hey, i just noticed that my browser shows me a little alert symbol in the Locksymbol.

![Bildschirmfoto 2021-03-16 um 14 24 17](https://user-images.githubusercontent.com/64002240/111317480-8029b280-8664-11eb-97b6-4bd78005120f.png)

I am using Firefox 86.0.1(64-Bit) on macOS 10.13.16

**Comment by blueonyx** [comment]
yep, there are 36 images linked via *http* instead of *https*, according to

```
grep -r "src=\"http://" fablabcb.github.io/
```

some are served from fablab-cottbus.de, so we could simply append the missing *s*, but i didn't check all images yet.

**Comment by blueonyx** [comment]
```
grep -r "src=\"http://blog.fablab-cottbus.de" fablabcb.github.io/|wc -l
```

seems to confirm that indeed all 36 images are hosted on our own blog.fablab-cottbus.de

**Comment by blueonyx** [comment]
there are also background images in style tags:

`grep -r "url('http://" .`
